app/models/world_location_email_signup.rb

```diff
@@ -0,0 +1,80 @@
+class WorldLocationEmailSignup
+  attr_reader :feed
+
+  def initialize(feed)
+    @feed = feed
+  end
+
+  def save
+    return false unless valid?
+
+    ensure_subscriber_list_exists
+    true
+  end
+
+  def signup_url
+    ensure_subscriber_list_exists["subscriber_list"]["subscription_url"]
+  end
+
+  def topic_id
+    ensure_subscriber_list_exists["subscriber_list"]["gov_delivery_id"]
```

Comment:
Is there a more modern field we can use instead of gov_delivery_id it feels rather legacy?

Reply:
Tbh I've just removed this bit of the code as it's not used and I'm not sure why I pulled it across from the original PR.